src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -576,8 +576,14 @@ SummaryConfig::SummaryConfig( const Deck& deck,
                               ErrorGuard& errors,
                               const GridDims& dims) {
     SUMMARYSection section( deck );
-    for( auto& x : section )
-        handleKW( this->keywords, x, schedule, tables, parseContext, errors, dims);
+    auto kw_iter = section.begin();
+    if (kw_iter != section.end())
+        kw_iter++;
```

Comment:
Maybe add a comment here that this is to skip `SUMMARY`?

Reply:
done

Reply:
Looks good.  Thanks!